examples/js/archive.js

```diff
@@ -89,8 +197,10 @@ const renderImages = (fullResImages, url) => {
 // renders thumbnails or images in thumbnail size
 const renderThumbnails = (thumbnails = [], url, fullResImgs) => {
   const imagesToRender = thumbnails || fullResImgs;
+  const paginatedImagess = paginate(imagesToRender);
+  imageCount = imagesToRender.length;
 
-  imagesToRender.forEach((file) => {
+  paginatedImagess[currPage].forEach((file) => {
```

Comment:
is imagess with 2 "s" a typo, or does it signify something? Thank you!

Reply:
It's a typo, will address it my next commit, thank you